tests/product/base_product_case.py

```diff
@@ -148,6 +149,20 @@ def setup_cluster(self, bare_image_provider, cluster_type):
             else:
                 raise RuntimeError("Docker images have not been created")
 
+    @staticmethod
+    def run_installers(cluster, installers, testcase):
```

Comment:
Oh, we use it in image_builder. Please comment that this shouldn't be used directly in test cases.